src/background/engineManager.ts

```diff
@@ -544,6 +548,41 @@ export class EngineManager {
     }
     return "ok";
   }
+
+  /**
+   * サードパーティ向けの設定ファイルを書き出す
+   */
+  async makeEngineInfoFor3rdParty() {
```

Comment:
どちらかというとwriteかも？
（まあmakeもするのですが、本当はmakeは他の関数がやって、この関数はドキュメントの通り書き込むだけが理想かな～と）
```suggestion
  /**
   * サードパーティ向けの設定ファイルを書き出す
   */
  async writeEngineInfoFor3rdParty() {
```